drivers/sensor/sensor_shell.c

```diff
@@ -106,6 +113,60 @@ static const char *sensor_attribute_name[SENSOR_ATTR_COMMON_COUNT] = {
 	[SENSOR_ATTR_FF_DUR] = "ff_dur",
 };
 
+/* Forward declaration */
+static void data_ready_trigger_handler(const struct device *sensor,
+				       const struct sensor_trigger *trigger);
+
+/**
+ * @brief This table stores a mapping of string trigger names along with the sensor_trigger struct
+ * that gets passed to the driver to enable that trigger, plus a function pointer to a handler. If
+ * that pointer is NULL, this indicates there is not currently support for that trigger type in the
+ * sensor shell.
+ */
+static const struct sensor_shell {
```

Comment:
nit: maybe consider a more specific name `sensor_shell_trigger_data`?

Reply:
I just made the struct type anonymous since we don't need a named type for this anyways.